07_VirtualFileSystem/03_TarFileSystem.md

```diff
@@ -0,0 +1,158 @@
+# The Tar File System
+
+In the previous chapter we have implemented the VFS layer. that will provide us with a common interface to access files and directories across different file systems on different devices. In this chapter we are going to implement our first file system and try to access it's content using the VFS function. As already anticipated we will implement the (US)TAR format.
```

Comment:
that -> That (new sentence starts with capital).

I would also maybe not say first file system, but first file system driver.